tfhe/src/shortint/server_key/mod.rs

```diff
@@ -258,6 +258,28 @@ pub struct ServerKey {
     pub pbs_order: PBSOrder,
 }
 
+impl ServerKey {
+    pub fn conformance_params(&self) -> CiphertextConformanceParams {
+        let lwe_dim = match self.pbs_order {
+            PBSOrder::KeyswitchBootstrap => self.key_switching_key.input_key_lwe_dimension(),
+            PBSOrder::BootstrapKeyswitch => self.key_switching_key.output_key_lwe_dimension(),
+        };
+
+        let ct_params = LweCiphertextParameters {
+            lwe_dim,
+            ct_modulus: self.ciphertext_modulus,
+        };
+
+        CiphertextConformanceParams {
+            ct_params,
+            message_modulus: self.message_modulus,
+            carry_modulus: self.carry_modulus,
+            degree: Degree(self.message_modulus.0 - 1),
```

Comment:
do we have an issue to plan to support a range of valid degree ?

Reply:
Yes https://github.com/zama-ai/tfhe-rs-internal/issues/270

Reply:
perfect thanks